docs/design/depleted_asset_protection.mdx

```diff
@@ -10,18 +12,15 @@ Ammalgam is an automated market maker that lends out unutilized assets. As price
 
 Below are some examples of how the UST de-pegging exhausted reserves on Uniswap V3 UST/USDC pool and emptied the Binance UST/USDT book. These are shared to point out that this condition is not unique to Ammalgam’s design, but has happened in other DeFi protocols and centralized exchanges. For Ammalgam, it is critical to know what the market thinks each asset is worth at all times in order to appropriately value collateral for outstanding loans.
 
-<blockquote class="twitter-tweet"><p lang="en" dir="ltr">1/ Fascinating little phenomenon happening in the Uniswap UST/USDC pool right now... The price in the pool is basically &quot;stuck&quot; at $0.952, despite the actual market price being $0.93. <a href="https://t.co/i6MQW7ubTd">pic.twitter.com/i6MQW7ubTd</a></p>&mdash; Doug Colkitt (@0xdoug) <a href="https://twitter.com/0xdoug/status/1523782169276997633?ref_src=twsrc%5Etfw">May 9, 2022</a></blockquote> 
-
-<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Binance book is empty. Never seen that before <a href="https://t.co/ez4ZZT9sDR">pic.twitter.com/ez4ZZT9sDR</a></p>&mdash; Hasu⚡️🤖 (@hasufl) <a href="https://twitter.com/hasufl/status/1523817151471230976?ref_src=twsrc%5Etfw">May 10, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-
-<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+<Tweet id="1523782169276997633" />
 
+<Tweet id="1523817151471230976" />
```

Comment:
I feel dumb, nice work cleaning this up.

Reply:
Nah, all good. It wouldn't have worked with Docusaurus v2 anyway as it was still using React v17 but the `react-tweet` required React v18